tests/proofs/dexter/dexter-compiled.md

```diff
@@ -0,0 +1,2928 @@
+                           PAIR ;
+                           SWAP ;
+                           DUP ;
+                           DUG 2 ;
+                           SWAP ;
+                           DUP ;
+                           CDR ;
+                           SWAP ;
+                           CAR ;
+                           SWAP ;
+                           DUP ;
+                           CDR ;
+                           SWAP ;
+                           CAR ;
+                           DIG 3 ;
+                           DUP ;
+                           DUG 4 ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CAR ;
+                           CONTRACT %transfer (list (pair address (list (pair address (pair nat nat))))) ;
+                           IF_NONE { PUSH nat 0 ; FAILWITH } {} ;
+                           PUSH mutez 0 ;
+                           NIL (pair address (list (pair address (pair nat nat)))) ;
+                           NIL (pair address (pair nat nat)) ;
+                           DIG 5 ;
+                           DIG 7 ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CDR ;
+                           CAR ;
+                           PAIR ;
+                           DIG 5 ;
+                           PAIR ;
+                           CONS ;
+                           DIG 4 ;
+                           PAIR ;
+                           CONS ;
+                           TRANSFER_TOKENS ;
+                           SWAP ;
+                           NIL operation ;
+                           DIG 2 ;
+                           CONS ;
+                           PAIR } } } } }
+       [macro]
+```
+
+# Translation
+
+To support the verbatim representation, we need to add some syntactic sugar.
+The parsed internal representation of Michelson contains nodes like explicit empty lists, e.g. `.AnnotationList`, and tokens to be wrapped using the `#token(...)` production.
+Here we implement a simplified parser that introduces what is necessary to paste the contract code directly.
+All the rules are macros, and they insert the `.AnnotationList` and `#token(...)` elements where necessary.
+
+# Instructions
+
+## Pretty Nullary Instructions
+
+```k
+  syntax Instruction ::= "DROP"
+                       | "DUP"
+                       | "SWAP"
+                       | "SOME"
+                       | "UNIT"
+                       | "PAIR"
+                       | "UNPAIR"
+                       | "CAR"
+                       | "CDR"
+                       | "CONS"
+                       | "SIZE"
+                       | "MEM"
```

Comment:
Let's move this to a file in the repo root called `pretty-instructions.md` or something, in its own module, then just import that module here.

Reply:
Done